x-pack/plugins/cases/common/schema/index.ts

```diff
@@ -153,3 +153,26 @@ export const limitedNumberSchema = ({ fieldName, min, max }: LimitedSchemaType)
       }),
     rt.identity
   );
+
+export interface RegexStringSchemaType {
```

Comment:
nit: does this need to be exported? ctrl + f doesn't show other occurrences.

Reply:
Nope!